examples/cms-dotcms/pages/_document.js

```diff
@@ -0,0 +1,15 @@
+import Document, {Html, Head, Main, NextScript} from 'next/document'
```

Comment:
This file should be removed, as it does not customize anything

Reply:
Removed.